Resources/Prototypes/_Sunrise/Surgery/amputation_steps.yml

```
@@ -16,3 +16,70 @@
+        types:
+          Slash: 15
+    - type: SurgeryStepAmputationEffect
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepInciseCartilage
+  name: Incise Cartilage
+  components:
+  - type: SurgeryStep
+    tools:
+      - type: Scalpel
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)